public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Calendar</title>
+</head>
+<body>
+  <div id="app"></div>
```

Comment:
Do you want to title this div/your parent component with your service name, instead of 'app'?  Just for purposes of when we put them all together (though if no one else uses app, it's fine).

Reply:
yea i'll change it. thanks.